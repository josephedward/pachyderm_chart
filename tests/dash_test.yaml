templates:
  - dash-deployment.yaml
  - dash-service.yaml
tests:
  # or you can snapshot the whole manifest
  - it: manifest should match snapshot
    asserts:
      - matchSnapshot: {}