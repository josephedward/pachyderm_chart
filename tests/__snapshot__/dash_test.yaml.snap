manifest should match snapshot:
  1: |
    apiVersion: apps/v1
    kind: Deployment
    metadata:
      labels:
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: pach_chart
        app.kubernetes.io/version: 0.5.48
        helm.sh/chart: pach_chart-0.1.0
      name: RELEASE-NAME-pach_chart
    spec:
      replicas: 1
      selector:
        matchLabels:
          app.kubernetes.io/instance: RELEASE-NAME
          app.kubernetes.io/name: pach_chart
      template:
        metadata:
          labels:
            app.kubernetes.io/instance: RELEASE-NAME
            app.kubernetes.io/name: pach_chart
        spec:
          containers:
          - image: pachyderm/dash:0.5.48
            imagePullPolicy: IfNotPresent
            livenessProbe:
              httpGet:
                path: /
                port: dash-http
            name: pach_chart-dash
            ports:
            - containerPort: 8080
              name: dash-http
              protocol: TCP
            readinessProbe:
              httpGet:
                path: /
                port: dash-http
            resources: {}
            securityContext: {}
          - image: pachyderm/grpc-proxy:0.4.10
            imagePullPolicy: IfNotPresent
            livenessProbe:
              httpGet:
                path: /
                port: grpc-proxy-http
            name: pach_chart-grpcProxy
            ports:
            - containerPort: 8081
              name: grpc-proxy-http
              protocol: TCP
            readinessProbe:
              httpGet:
                path: /
                port: grpc-proxy-http
            resources: {}
            securityContext: {}
          securityContext: {}
          serviceAccountName: RELEASE-NAME-pach_chart
  2: |
    apiVersion: v1
    kind: Service
    metadata:
      labels:
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: pach_chart
        app.kubernetes.io/version: 0.5.48
        helm.sh/chart: pach_chart-0.1.0
      name: RELEASE-NAME-pach_chart
    spec:
      ports:
      - name: dash-http
        nodePort: 30080
        port: 8080
      - name: grpc-proxy-http
        nodePort: 30081
        port: 8081
      selector:
        app: dash
        suite: pachyderm
      type: ClusterIP
